sdk/communication/Azure.Communication.Rooms/CHANGELOG.md

```diff
@@ -0,0 +1,4 @@
+# Release History
+
+## 1.0.0-beta.1 (Unreleased)
```

Comment:
@elavarasidc is this going to beta still?

Reply:
Yes, since we are Pre GA, we will be releasing as beta.